substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -250,10 +255,19 @@ where
 		},
 	};
 
-	let module = polkavm::Module::from_blob(&engine, &polkavm::ModuleConfig::default(), blob)?;
-	let mut linker = polkavm::Linker::new(&engine);
+	let module =
+		polkavm::Module::from_blob(&engine, &polkavm::ModuleConfig::default(), blob.clone());
+	let module = match module {
+		Ok(ref module) => module,
+		Err(ref error) => Err(WasmError::Other(error.to_string()))?,
+	};
```

Comment:
Add a `From` impl to make `?` work and use `?` here to propagate the error message.

Reply:
Will do (and address other error handling comments).